opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -104,9 +106,175 @@ public void test_zero_terms_query_parameter_lower_case() {
     Assertions.assertNotNull(matchPhraseQuery.build(new MatchPhraseExpression(arguments)));
   }
 
+  @Test
+  public void test_SyntaxCheckException_when_no_arguments_match_phrase_syntax() {
+    List<Expression> arguments = List.of();
+    assertThrows(SyntaxCheckException.class,
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseWithUnderscoreName)));
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_one_argument_match_phrase_syntax() {
+    List<Expression> arguments = List.of(dsl.namedArgument("field", "test"));
+    assertThrows(SyntaxCheckException.class,
+        () -> matchPhraseQuery.build(new MatchPhraseExpression(
+            arguments, MatchPhraseQueryTest.this.matchPhraseWithUnderscoreName)));
+
+  }
+
+  @Test
+  public void test_SyntaxCheckException_when_invalid_parameter_match_phrase_syntax() {
+    List<Expression> arguments = List.of(
+        dsl.namedArgument("field", "test"),
```

Comment:
Please use `DSL.namedArgument`. Either works now, but `dsl.namedArgument` is going away immanently upstream.

Reply:
This has been fixed